tests/app/dao/test_annual_billing_dao.py

```diff
@@ -0,0 +1,35 @@
+from app.service.utils import get_current_financial_year_start_year
+from app.models import AnnualBilling
+from app.dao.annual_billing_dao import (
+    dao_create_or_update_annual_billing_for_year,
+    dao_get_free_sms_fragment_limit_for_year
+)
+
+
+def test_sample_service_has_free_sms_fragment_limit(notify_db_session, sample_service):
```

Comment:
Its difficult to see what methods are being tested. i'd rename to the test_<the dao method you are testing>_<extra info>

e.g. test_dao_get_free_sms_fragment_limit_for_year_for_sample_service